src/services/SyncService.js

```diff
@@ -270,7 +270,7 @@ class SyncService {
 	async close() {
 		this.backend?.disconnect()
 		const timeout = new Promise((resolve) => setTimeout(resolve, 2000))
-		await Promise.any([timeout, this.save()])
+		await Promise.any([timeout, this.save({ manualSave: false })])
```

Comment:
Shouldn’t the flag then be true here?

Reply:
Ah this is the default, then I don't get why turning off the manual save flag would fix this as now we would hit the `AUTOSAVE_MINIMUM_DELAY` https://github.com/nextcloud/text/blob/3b7877b14e4dc0b4decb2a5d212d15dd22b5d031/lib/Service/DocumentService.php#L343

Reply:
> Shouldn’t the flag then be true here?

Err… absolutely

Reply:
No, something's not right here. Seems I also got the repro steps not as reliable as is should, especially manualSave was even by default set to true.

Reply:
Now that's embarrassing. I cannot reproduce the original issue at all anymore. The `false` must have been a mistake from me yesterday due to cleaning up (had it at a different position first, plus debug notes). Anyway, embarrassing.

Reply:
Yeah, I think reproduction steps are loose and probably depend a lot on timing and if there were previous steps added to the document, so could well be that on the first look it seemed like the change worked.

The separate save request might not help directly, but would allow us to better observe what is happening in failure cases, make cypress tests more reliable as we could await the save request and maybe also helps with creating reproduction test cases later on where we artificially block individual requests upfront to simulate timing problems.